pkg/parser/mysql/const.go

```diff
@@ -191,6 +191,9 @@ const (
 const (
 	// SystemDB is the name of system database.
 	SystemDB = "mysql"
+	// SysDB is the name of `sys` schema, which is a set of objects to help users to interpret data collected
+	// in `information_schema`.
+	SysDB = "sys"
```

Comment:
This variable name is a little strange, and is confusing considering the existing `SystemDB` constant 🤦 . Does anyone have better suggestion?